code/modules/unit_tests/_unit_tests.dm

```diff
@@ -74,6 +74,7 @@
 /// A trait source when adding traits through unit tests
 #define TRAIT_SOURCE_UNIT_TESTS "unit_tests"
 
+#include "atmospherics/_atmospherics.dm"
```

Comment:
include a file that includes a file, that is empty?

Reply:
fixed